### PR TITLE
fix for more than 255 leds + map.h generator script

### PR DESCRIPTION
**Pull request by mariusmotea**

### CODE REVIEW

**Comment by Wqrld** [comment]
as the project seems to be dead, can you make a fork to revive it mariusmotea?

**Comment by mariusmotea** [comment]
We need to wait more, maybe the guy is on vacation.

**Comment by jasoncoon** [comment]
Yeah, seriously Wqrld.  A guy goes offline for a few days, and the project is dead?  :)

**Comment by jasoncoon** [comment]
Thanks for the contribution!  I'm away on business, will review, test, and merge this as soon as I can.

**Comment by mariusmotea** [comment]
For sure the number of leds is uint8_t in other places, but you must have the code in your mind and will be able to replace with uint16_t. I'm thinking to add support in my diyHue project.

**Comment by Wqrld** [comment]
Very sorry on the deadline project thingy, I concluded that because there were no contribs in 6 months :)

File: Map.h [hunk 1]
```diff
@@ -16,22 +16,22 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-const uint8_t radii[NUM_LEDS] = {
+const uint16_t radii[NUM_LEDS] = {
```
Comment (jasoncoon) [inline Map.h]: Some of these dimensions in Map.h are meant to be a uint8_t (0 to 255) range, regardless of the number of LEDs:

* radii
* conicalRadii
* angles
* levels
* xCoords
* yCoords
* zCoords

I'm sure they work as uint16_t, but will needlessly consume more RAM.

File: Meteors.h [hunk 2]
```diff
@@ -38,8 +38,8 @@ void whiteMeteors() {
   dimAll(220);
 
   EVERY_N_MILLISECONDS(30) {
-    for (uint8_t meteorIndex = 0; meteorIndex < meteorCount; meteorIndex++) {
-      uint8_t i = getMeteorLedIndexAndUpdate(meteorIndex);
+    for (uint16_t meteorIndex = 0; meteorIndex < meteorCount; meteorIndex++) {
```
Comment (jasoncoon) [inline Meteors.h]: meteorIndex can stay uint8_t, unless you plan on changing meteorCount as well, and having more than 255 meteors.  :)

Comment (mariusmotea) [reply Meteors.h]: Hi, this is one of the current problem i have. Meteors are not placed on top of the tree. I will update the PR.

Comment (jasoncoon) [reply Meteors.h]: I was never very satisfied with how the meteors animations turned out, almost pulled them completely.

**Comment by jasoncoon** [comment]
Thank you for your contributions!

I've made some nice improvements with the Arduino and web apps in v1.1 of my esp8266-fastled-webserver project.  I would like to bring those over to this and other projects: https://github.com/jasoncoon/esp8266-fastled-webserver/tree/v1.1

**Comment by thesnowpup** [comment]
Hi Jason.

Did you ever manage to merge across your improvements to this project?

Thank you for all your hard work.

**Pull request merged**